### ISSUE TITLE
Cryptocurrencies are lowercase

### ISSUE THREAD

**Issue by tamkar1982** [Howchoo/crypto-ticker#7]
Is it possible to get uppercase letters for tickersymbols?

**Comment by josephtyler** [comment]
Yes it is. I was thinking about how to fix this. Do you think it would make sense for the case to just match whatever you use in the settings.env file?

**Comment by tamkar1982** [comment]
I don't know what will be the best solution, but in my opinion ticker symbols should always be in uppercase letters. Maybe set this to a globally setting?

**Comment by DNA2RNA1** [comment]
Was there any solution to this issue? Uppercase symbols in the setting.env file are not recognized.  Would be awesome if the display matched the case in the setting.env file or Uppercase was the default for display.

**Comment by tamkar1982** [comment]
Hi DNA2RNA1,

I have solved it in the "price_apis.py" file.

Just edit the file with a ".upper()" method in the marked line (see screenshot).

![Bildschirmfoto 2021-05-30 um 21 53 42](https://user-images.githubusercontent.com/80121234/120118291-06e52a80-c192-11eb-8857-ec9ee89709c2.png)

**Comment by DNA2RNA1** [comment]
Thank you for the solution!
